stackslib/src/chainstate/stacks/boot/mod.rs

```diff
@@ -1660,11 +1660,8 @@ pub mod test {
         lock_period: u128,
         burn_ht: u64,
     ) -> StacksTransaction {
-        // ;; TODO: add signer key
-        // (define-public (stack-stx (amount-ustx uint)
-        //                           (pox-addr (tuple (version (buff 1)) (hashbytes (buff 32))))
-        //                           (burn-height uint)
-        //                           (lock-period uint))
+        let public_key = StacksPublicKey::from_private(key);
```

Comment:
This `key` is the key used to sign the transaction, not the DKG signing key.  There needs to be a separate argument to `make_pox_4_lockup` for that signing key.